.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build/release
+
+on: push
```

Comment:
Maybe we should build the APK only on tag releases and not on commits, what do you think?

Reply:
By keeping it this way, we are able to track possible problems with the `electron-build` pipeline.
The action installs NPM dependencies, runs the build script (`yarn build`), and then packages the Electron app.
It will only create a GitHub release on correctly tagged commits `v*.*.*`.

Reply:
@ruisaraiva19 Does the electron-builder action automatically install dependencies? Is there any way for us to provide a cache to it using `yarn.lock`?

Reply:
@diego3g yes, the action runs `yarn install` out of the box.
We can add the `actions/cache` action to cache the [yarn cache directory](https://github.com/actions/cache/blob/master/examples.md#node---yarn) after `actions/setup-node` action and before the `samuelmeuli/action-electron-builder` action.

Reply:
I pushed [a commit](https://github.com/diego3g/rocketredis/pull/39/commits/bb69f24ba75134f96940c310d96b97749d2a72cb) to cache the yarn cache directory.